### DISCUSSION TITLE
How to extend payload limit, any example code

### DISCUSSION THREAD

**Discussion by devanshsinghvaluecoders** [vercel/vercel#6431] [Help]
I want to extend payload limit but didn't get any example from which i can learn

**Comment by mcsdevv** [comment]
Hi @devanshsinghvaluecoders, the [payload limit](https://vercel.com/docs/platform/limits#serverless-function-payload-size-limit) is not something that can be extended - can you share more about your use case?

**Reply by devanshsinghvaluecoders** [reply]
Use azure server for uploading data,working perfectly fine in local but when i deployed it on vercel , and try to upload file greater than 5mb than got 413 error (payload extend).
